src/SSVOpenHexagon/Components/CWall.cpp

```diff
@@ -55,23 +55,34 @@ void CWall::update(HexagonGame& mHexagonGame, ssvu::FT mFT)
 void CWall::moveTowardsCenter(
     HexagonGame& mHexagonGame, const sf::Vector2f& mCenterPos, ssvu::FT mFT)
 {
+    const float wallSpawnDist{mHexagonGame.getLevelStatus().wallSpawnDistance};
+    const float wallAccSpawnDist{mHexagonGame.getLevelStatus().wallAccSpawnDistance};
+
     const float radius{mHexagonGame.getRadius() * 0.65f};
+    const float outerBounds =
+        (wallSpawnDist == wallAccSpawnDist) ? wallSpawnDist * 0.5f : 800.f;
```

Comment:
Please add a comment here, why the hardcoded `800.f` and comparison between the two distances?